extensions/positron-assistant/package.json

```diff
@@ -110,18 +110,6 @@
         "title": "%commands.configureModels.title%",
         "category": "%commands.category%",
         "enablement": "config.positron.assistant.enable"
-      },
-      {
```

Comment:
This is just the part to define the commands so they show up in the command palette. The extension itself calls the commands and that can be refactored to not have to use the commands anymore.